channels/serializers.py

```diff
@@ -229,6 +231,14 @@ def get_upvoted(self, instance):
         """Is a comment upvoted?"""
         return instance.likes is True
 
+    def get_profile_image(self, instance):
+        """Find the Profile for the comment author"""
+        profile = self.context['request'].user.profile
```

Comment:
This `user` is the user viewing the API, not the user for the profile

Reply:
yeah I just fixed that